perkeeppy/uploadhelper.py

```diff
@@ -1,4 +1,8 @@
-import io
+# -*- coding: utf-8 -*-
+
+import requests
+
+from .exceptions import ServerError
```

Comment:
Please use absolute imports

```suggestion
from perkeeppy.exceptions import ServerError
```